janusgraph-cql/src/main/java/org/janusgraph/diskstorage/cql/CQLConfigOptions.java

```diff
@@ -285,4 +285,44 @@
             ConfigOption.Type.MASKABLE,
             String.class);
 
+    // Query logger
+    ConfigNamespace QUERY_LOGGER = new ConfigNamespace(
+        CQL_NS,
+        "query-logger",
+        "CQL query logger options");
+
+    ConfigOption<Boolean> QUERY_LOGGER_ENABLED = new ConfigOption<>(
+        QUERY_LOGGER,
+        "enabled",
+        "Registers a query logger with datastax cassandra driver",
+        ConfigOption.Type.LOCAL,
+        false);
+
+    ConfigOption<Integer> QUERY_LOGGER_MAX_QUERY_STRING_LENGTH = new ConfigOption<>(
+        QUERY_LOGGER,
+        "max-query-string-length",
+        "Max query string length to be logged when query logger is enabled. Use '-1' to use datastax default value.",
+        ConfigOption.Type.LOCAL,
+        -1);
```

Comment:
It would be okay to have on default value.

Reply:
https://github.com/JanusGraph/janusgraph/blob/master/janusgraph-core/src/main/java/org/janusgraph/graphdb/configuration/GraphDatabaseConfiguration.java#L394